### PR TITLE
fix: parse jcloud labels as str

### PR DESCRIPTION
**Pull request by npitsillos**
**Goal**

- Parses the flow dict loaded with `JAML.load` and converts the change types of `jcloud.labels` from an `int` to `str`

- [x] Run [Integration tests GHA](https://github.com/jina-ai/jcloud/actions/workflows/integration-tests.yml) manually & comment the link. https://github.com/jina-ai/jcloud/actions/runs/5180469639

@jina-ai/team-wolf

### CODE REVIEW

File: jcloud/normalize.py [hunk 1]
```diff
@@ -39,6 +39,35 @@ class FlowYamlNotFound(FileNotFoundError):
...
 
 
+def stringify(v: Any) -> str:
+    if isinstance(v, str):
+        return v
+    elif isinstance(v, int) or isinstance(v, float):
+        return str(v)
+
+
+def stringify_labels(flow_dict: Dict) -> Dict:
```
Comment (deepankarm) [inline jcloud/normalize.py:49]: Pls add unit tests for this.

Comment (npitsillos) [reply jcloud/normalize.py:49]: Just added one let me know if this will suffice.

File: jcloud/normalize.py [hunk 2]
```diff
@@ -39,6 +39,36 @@ class FlowYamlNotFound(FileNotFoundError):
     pass
 
 
+def stringify(v: Any) -> str:
+    if isinstance(v, str):
+        return v
+    elif isinstance(v, int) or isinstance(v, float):
+        return str(v)
```
Comment (tarrantro) [inline jcloud/normalize.py:46]: The value type is Any, but your coditions only have int/float and str. maybe you can print the error for other types?

File: tests/unit/flows/flow-with-labels.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+jtype: Flow
+jcloud:
+  labels:
+    test-label-one: 1
+    test-label-two: 231
```
Comment (tarrantro) [inline tests/unit/flows/flow-with-labels.yml:5]: can you add few more test case for invalid output? like:
```
jcloud:
  labels:
    test-label-complex: 1j
```

```
jcloud:
  labels:
    test-label-invalid-obj: 
       foo: bar 
```

```
jcloud:
  labels:
    test-label-bool:  true
```

**Review by tarrantro — APPROVED** [review]
lgtm

**Pull request merged**